docs/app/pages/[...slug].vue

```diff
@@ -135,8 +135,10 @@ const communityLinks = computed(() => [{
           :target="link.to.startsWith('http') ? '_blank' : undefined"
           v-bind="link"
         >
+          <span class="sr-only">Go to {{ link.label }}</span>
```

Comment:
Is this really necessary? 🤔

Reply:
![CleanShot 2025-03-25 at 11 08 12@2x](https://github.com/user-attachments/assets/27941102-9209-4f0b-9640-4d5bf9dabdc5)